packages/node/features/sessions.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Server side session tracking
```

Comment:
It may be worth considering a scenario that tests batch delivery. For example, the Java notifier deliver sessions in a batch every X seconds, and I believe that we have a test that ensures that if 100 sessions are created over that boundary, M sessions are sent in the first request, and M sessions are sent in the second. I don't know if that's already covered or if a similar mechanism is used here.

Reply:
Good point, I think this would be a decent addition. Let me see if I can knock something together quickly when I address the other feedback, and if not I'll make a ticket to add this at a later point.